coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/query/GetCommitsInProjectController.java

```diff
@@ -22,20 +20,12 @@ public GetCommitsInProjectController(GetCommitsInProjectUseCase getCommitsInProj
     this.getCommitsInProjectUseCase = getCommitsInProjectUseCase;
   }
 
-  @GetMapping(path = "/projects/{projectId}/commits", produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<List<GetCommitResponse>> listCommits(
-      @PathVariable("projectId") Long projectId) {
-    List<Commit> commits = getCommitsInProjectUseCase.get(projectId);
-    List<GetCommitResponse> responses = new ArrayList<>(commits.size());
-    for (Commit commit : commits) {
-      responses.add(
-          new GetCommitResponse()
-              .setName(commit.getName())
-              .setAnalyzed(commit.isAnalyzed())
-              .setAuthor(commit.getAuthor())
-              .setComment(commit.getComment())
-              .setTimestamp(commit.getTimestamp()));
-    }
-    return new ResponseEntity<>(responses, HttpStatus.OK);
+  @GetMapping(
+      path = "/projects/{projectId}/{branchName}/commits",
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<List<Commit>> listCommits(
```

Comment:
`GetCommitResponse` is no longer used anymore. We can remove this class and replace its occurrences in the tests with the `Commit` class.